pilot/pkg/serviceregistry/kube/controller/serviceexportcache.go

```diff
@@ -112,6 +111,10 @@ type serviceExportCacheImpl struct {
 
 	// clusterSetLocalPolicySelector selects an appropriate EndpointDiscoverabilityPolicy for the clusterset.local host.
 	clusterSetLocalPolicySelector discoverabilityPolicySelector
+
+	serviceExportCh chan struct{}
+
+	started atomic.Bool
```

Comment:
is this needed now?

Reply:
Yes, i need it to protect calling ec.filteredInformer nil pointer panic

I can check `if ec.filteredInformer` either, but will need another lock to prevent data race